site/docs/4.1/components/spinners.md

```diff
@@ -10,14 +10,16 @@ toc: true
 
 Bootstrap "spinners" can be used to show the loading state in your projects. They're built only with HTML and CSS, meaning you don't need any JavaScript to create them. You will, however, need some custom JavaScript to toggle their visibility. Their appearance, alignment, and sizing can be easily customized with our amazing utility classes.
 
-For accessibility purposes, each loader here includes `role="status"` and `Loading...` text within. We account for this in our CSS, using a text hiding technique to prevent it from rendering on screen.
+For accessibility purposes, each loader here includes `role="status"` and `<span class="sr-only">Loading...</span>` within.
```

Comment:
the "...within" bit reads a bit awkward. Perhaps turn the sentence around to "and a nested `<span class="sr-only">Loading...</span>`."